DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -272,9 +272,11 @@
   <ioread sourceClass = "pat::Jet" version="[1-]" targetClass="pat::Jet" source="" target="daughtersTemp_">
     <![CDATA[daughtersTemp_.reset();]]>
   </ioread>
-  <class name="pat::MET"  ClassVersion="15">
-    <version ClassVersion="15" checksum="428901429"/>
+  <class name="pat::MET"  ClassVersion="17">
+    <version ClassVersion="17" checksum="2416242778"/>
```

Comment:
Please use consecutive numbers for ClassVersions, here "16".

Reply:
done